src/ui/i18n/en-US.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Yeah, this is just a test file that got added to the commit by mistake. I will remove it on next commit.